src/Application/Messenger/Handler/StripePayoutHandler.php

```diff
@@ -145,10 +147,10 @@ public function __invoke(StripePayout $payout): void
                 // method is called only when Stripe event `payout.paid` is received.
                 $donation->setDonationStatus('Paid');
 
+                $donation->setSalesforcePushStatus(SalesforceWriteProxy::PUSH_STATUS_PENDING_UPDATE);
```

Comment:
As we're adding a call to it it might be worth narrowing down the param type on `setSalesforcePushStatus` like so:

```php
    /**
     * @psalm-param self::PUSH_STATUS_* $salesforcePushStatus // <--------- new line
     * @param string $salesforcePushStatus
     */
    public function setSalesforcePushStatus(string $salesforcePushStatus): void
    ````